kmod/src/Makefile.kernelcompat

```diff
@@ -169,3 +169,10 @@ endif
 ifneq (,$(shell grep 'int...get..const struct xattr_handler.*struct dentry.*dentry,' include/linux/xattr.h))
 ccflags-y += -DKC_XATTR_STRUCT_XATTR_HANDLER=1
 endif
+
+#
+# Network stack changes with v4.x
```

Comment:
There's too many unrelated interface changes in here.  They should be broken up and triggered by actual interface changes, not presumed association with the presence of the user_msghdr.